resources/public/api/conf/1.0/schemas/retrieve_relief_investments_response.json

```diff
@@ -0,0 +1,220 @@
+            "type": "number",
+            "example": "2000.99"
+          }
+        }
+      }
+    },
+    "seedEnterpriseInvestment": {
+      "description": "Financial details about SEED investment income.",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "uniqueInvestmentRef": {
+            "description": "The Unique Investment Reference (UIR) or the name of the authorising tax office as shown on the certificate.",
+            "type": "string",
+            "example": "12345678"
+          },
+          "companyName": {
+            "description": "The name of the company holding the investment.",
+            "type": "string",
+            "example": "Goldman Sachs"
+          },
+          "dateOfInvestment": {
+            "description": "The date of the investment",
+            "type": "string",
+            "example": "2018-04-06"
+          },
+          "amountInvested": {
+            "description": "The total amount invested.",
+            "type": "number",
+            "example": "2000.99"
+          },
+          "reliefClaimed": {
+            "description": "The amount of relief being claimed for the tax year.",
+            "type": "number",
+            "example": "2000.99"
+          }
+        }
+      }
+    },
+    "socialEnterpriseInvestment": {
+      "description": "Financial details about social enterprise investment income.",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "uniqueInvestmentRef": {
+            "description": "The Unique Investment Reference (UIR) or the name of the authorising tax office as shown on the certificate.",
+            "type": "string",
+            "example": "12345678"
+          },
+          "socialEnterpriseName": {
+            "description": "The name of the investment or fund.",
+            "type": "string",
+            "example": "SE Inc"
+          },
+          "dateOfInvestment": {
+            "description": "The date of the investment.",
+            "type": "string",
+            "example": "2018-04-06"
+          },
+          "amountInvested": {
+            "description": "The total amount invested.",
+            "type": "number",
+            "example": "2000.99"
+          },
+          "reliefClaimed": {
+            "description": "The amount of relief being claimed for the tax year.",
+            "type": "number",
+            "example": "2000.99"
+          }
+        }
+      }
+    },
+    "links": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "description": "A list of endpoint links that indicate possible actions related to the current resource",
+        "properties": {
+          "href": {
+            "description": "The relative url of the endpoint",
+            "type": "string",
+            "example": "/individuals/reliefs/investment/{nino}/{taxYear}"
```

Comment:
Please replace `{nino}` and `{taxYear}` with valid examples (like you did in the example response JSON)